api/Element.json

```diff
@@ -4545,11 +4539,12 @@
               "version_added": "16"
             },
             "firefox": {
-              "version_added": true,
-              "notes": "[1] Before Firefox 35, it was implemented on the <code>Node</code> interface."
+              "version_added": "1",
+              "notes": "Before Firefox 35, it was implemented on the <code>Node</code> interface."
```

Comment:
Since `Element` inherits from `Node`, this note strikes me as not useful. But I'll not suggest changing it here. There are lots of notes I'd like to see changed all over, better keep the projects separate :)